public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.test.ts

```diff
@@ -0,0 +1,290 @@
+import { M } from 'msw/lib/glossary-dc3fd077';
+
+import LokiLanguageProvider from '../../../LanguageProvider';
+import { LokiDatasource } from '../../../datasource';
+import { createLokiDatasource } from '../../../mocks';
+
+import { CompletionDataProvider } from './CompletionDataProvider';
+import { getCompletions } from './completions';
+import { Label, Situation } from './situation';
+
+const history = [
+  {
+    ts: 12345678,
+    query: {
+      refId: 'test-1',
+      expr: '{test: unit}',
+    },
+  },
+  {
+    ts: 87654321,
+    query: {
+      refId: 'test-1',
+      expr: '{test: unit}',
+    },
+  },
+];
+
+const labelNames = ['place', 'source'];
+const labelValues = ['moon', 'luna'];
+const extractedLabelKeys = ['extracted', 'label'];
+const otherLabels: Label[] = [
+  {
+    name: 'place',
+    value: 'luna',
+    op: '=',
+  },
+];
+const afterSelectorCompletions = [
+  {
+    insertText: '|= "$0"',
+    isSnippet: true,
+    label: '|= "something"',
```

Comment:
We should probably change "something" to something else :D Maybe just `|= ""` is enough.

Reply:
Maybe even adding there documentation that it is line contains

Reply:
Good catch.

> Maybe even adding there documentation that it is line contains

Not sure I understand.

Reply:
Some functions/options have this additional explanation:
<img width="1296" alt="image" src="https://user-images.githubusercontent.com/30407135/194114914-89c364cb-b838-4572-a4d6-02b0159e1d4c.png">

So maybe for `|= ""` we could add explanation that it is 'line contains'. And `|~ ""` is regex match.